MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE
+recursive-include m2cgen VERSION.txt
```

Comment:
Why recursive?

Reply:
I think it is the syntax to include filed from directories. 

> 

recursive-include dir-pattern pat1 pat2 ... | Include all files under directories matching dir-pattern that match any of the listed patterns
-- | --




https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands

I can be wrong, but `include` works only for root-located files...